examples/bri-3/src/bri/auth/services/auth.service.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@nestjs/common';
+import { JwtService } from '@nestjs/jwt';
+import { AuthAccesorService } from '../agents/authAccessor.agent';
+import { errorMessage } from '../constants';
+import { Login } from '../model/login';
+import { LoginService } from './login.service';
+
+@Injectable()
+export class AuthService {
+  constructor(
+    private readonly authAccesorService: AuthAccesorService,
```

Comment:
we introduce 4 services that are at least somehow  connected
- auth accessor service
- jwt service
- auth service
- login service

why so many services for simple auth flow?

Reply:
No, they are not at all connected. Each service has its own defined responsibility.